### PR TITLE
SignalR 2.2.0

### PR DESCRIPTION
**Pull request by GiedriusGrabauskas**
### Hub connection interface

Before:

``` ts
interface MyHubConnection extends HubConnection { }
```

After:

``` ts
interface MyHubConnection extends SignalR.Hub.Connection { }
```
### SignalR connection interface

Before:

``` ts
var connection: SignalR;
```

After:

``` ts
var connection: SignalR.Connection;
```
### SignalR Hub Proxy interface

Before:

``` ts
var proxy: HubProxy;
```

After:

``` ts
var proxy: SignalR.Hub.Proxy;
```
### State changed callback

Before:

``` ts
function stateChanged = (state: SignalRStateChange) => { };
```

After:

``` ts
function stateChanged = (state: SignalR.StateChanged) => { };
```

### CODE REVIEW

**Comment by DovydasNavickas** [comment]
:+1:

**Comment by vvakame** [comment]
*angular-signalr-hub/angular-signalr-hub.d.ts*

to author (@AdamSantaniello). Could you review this PR?
:+1: or :-1:?

Checklist

* [ ] pass the Travis CI test?

File: signalr/signalr.d.ts [hunk 1]
```diff
@@ -1,113 +1,344 @@
-// Type definitions for SignalR 1.0
+// Type definitions for SignalR 2.2.0
 // Project: http://www.asp.net/signalr
-// Definitions by: Boris Yankov <https://github.com/borisyankov/>, T. Michael Keesey <https://github.com/keesey/>
```
Comment (vvakame) [inline signalr/signalr.d.ts]: please keep original authors name. add your name below.

Comment (DovydasNavickas) [inline signalr/signalr.d.ts]: It is a new version written from scratch. Does he have to keep original authors, even though the definitions were written looking at the source directly?

**Comment by AdamSantaniello** [comment]
:+1:

**Comment by vvakame** [comment]
I decided to merge this PR. thanks!

**Pull request merged**